myuw/dao/class_website.py

```diff
@@ -33,16 +33,11 @@ def get_page_title_from_url(url):
     try:
         html = _fetch_url('GET', url)
         soup = BeautifulSoup(html)
-        return soup.title.string
+        if hasattr(soup, 'title'):
```

Comment:
>>> from BeautifulSoup import BeautifulSoup
>>> soup = BeautifulSoup("<html>no title</html>")
>>> print hasattr(soup, 'title')
True
>>> print soup.title.string
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'string'
>>>